src/compiler/crystal/tools/doc/html/css/style.css

```diff
@@ -349,6 +349,43 @@ code {
   font-family: Consolas, 'Courier New', Courier, Monaco, monospace;
 }
 
+span.flag {
+  padding: 2px 4px 1px;
+  border-radius: 3px;
+  margin-right: 3px;
+  font-size: 11px;
```

Comment:
This came to mind but I i didn't think it was necessary. In hindsight it is better that way. Good eye. 👁 👍